shared/js/background/chrome-events.es6.js

```diff
@@ -716,9 +723,10 @@ if (chrome.webRequest.OnBeforeSendHeadersOptions.EXTRA_HEADERS) {
 chrome.webRequest.onBeforeSendHeaders.addListener(
     request => {
         const GPCHeader = GPC.getHeader()
+        const GPCEnabled = tdsStorage.config.features?.gpc.state === 'enabled'
 
         let requestHeaders = request.requestHeaders
-        if (GPCHeader) {
+        if (GPCHeader && GPCEnabled) {
```

Comment:
ditto, there was no way to disable GPC before?

Reply:
It was just the DOM signal before